src/app/tvshows/store/tvshows.reducer.spec.ts

```diff
@@ -0,0 +1,50 @@
+import * as tvshows from '../index';
+import { TvShowData } from './tvshows.models';
+
+describe('The tvshows feature', () => {
```

Comment:
Test should be named something like `TvShows Reducer`

Reply:
Yes, I will fix that.